tools/version-tracker/pkg/commands/upgrade/upgrade.go

```diff
@@ -258,28 +259,34 @@ func Run(upgradeOptions *types.UpgradeOptions) error {
 				updatedFiles = append(updatedFiles, projectReadmePath)
 
 				// Update the checksums file and attribution file(s) corresponding to the project.
-				if !projectHasPatches {
-					if _, err := os.Stat(filepath.Join(projectRootFilepath, constants.ChecksumsFile)); err == nil {
-						logger.Info("Updating project checksums and attribution files")
-						projectChecksumsFileRelativePath := filepath.Join(projectPath, constants.ChecksumsFile)
-						err = updateChecksumsAttributionFiles(projectRootFilepath)
-						if err != nil {
-							return fmt.Errorf("updating project checksums and attribution files: %v", err)
-						}
-						updatedFiles = append(updatedFiles, projectChecksumsFileRelativePath)
+				if projectHasPatches {
```

Comment:
If it fails to apply, do you want to try checking in the .rej files? im not sure how helpful they will be but maybe will provide some indication to how big the conflict is?

Reply:
We currently don't generate the .rej files when applying patches, but it's just a matter of adding a flag though. Are you saying we start doing that?

Reply:
The patch will need human intervention anyway so they're going to know when they apply it locally. I don't think it will make a big difference to check it in. Maybe in the patch comment, we can specify how many patches were applied before it failed, and what files had conflicts.

Reply:
Included failed patch number and files information (Example: https://github.com/abhay-krishna/eks-anywhere-build-tooling/pull/57#issuecomment-1989500413) in the warning.